src/app/components/sites/site/site.component.html

```diff
@@ -71,33 +86,35 @@ <h3>Recent Annotations</h3>
 
       <ul class="nav nav-pills">
         <!-- Loading animation -->
-        <baw-loading *ngIf="!taggings"></baw-loading>
+        <baw-loading *ngIf="!events" id="annotation-loader"></baw-loading>
 
         <!-- Handle no annotations -->
-        <p *ngIf="taggings?.length === 0">No recent annotations found</p>
+        <p *ngIf="events?.length === 0">No recent annotations found</p>
 
         <!-- For each annotation and tagging -->
-        <li *ngFor="let tagging of taggings" class="nav-item">
-          <!-- Annotation link -->
-          <a class="nav-link" [routerLink]="tagging.viewUrl">
-            <!-- Loading when unresolved -->
-            <ng-container
-              *ngIf="
-                (tagging.tag | isUnresolved) ||
-                  (tagging.tag?.creator | isUnresolved);
-                else loaded
-              "
-            >
-              <baw-loading size="sm"></baw-loading>
-            </ng-container>
+        <ng-container *ngFor="let event of events">
+          <li *ngFor="let tagging of event.taggings" class="nav-item">
+            <!-- Annotation link -->
+            <a class="nav-link" [routerLink]="event.listenViewUrl">
+              <!-- Loading when unresolved -->
+              <ng-container
+                *ngIf="
+                  (tagging.tag | isUnresolved) ||
+                    (tagging.tag?.creator | isUnresolved);
+                  else loaded
+                "
+              >
+                <baw-loading id="tag-loader" size="sm"></baw-loading>
+              </ng-container>
 
-            <!-- Annotation details -->
-            <ng-template #loaded>
-              "{{ tagging.tag?.text }}" by
-              {{ tagging.tag?.creator?.userName }}
-            </ng-template>
-          </a>
-        </li>
+              <!-- Annotation details -->
+              <ng-template #loaded>
+                "{{ tagging.tag?.text }}" by
+                {{ tagging.tag?.creator?.userName }}
```

Comment:
these should ideally be both links to the library list page that pre-populate the search filter. Too hard?

Reply:
The current implementation on the ecosounds website is to link to the listen page, not the library

Reply:
ah i missed the anchor in wrapper for the template. disregard